packages/learner-access/src/components/LoadUserLearning.tsx

```diff
@@ -175,7 +174,7 @@ const LoadUserLearning = ({ query, kind, sort }: LoadedComponentProps): JSX.Elem
           </div>
         </div>
         <div className=" archive mt-2 row-end-4 text-black-light relative">
-          {item.availabilityStatus !== 'completed' && (
+          {item.callToAction !== 'completed' && (
```

Comment:
I don't believe the Archive button is gated by the content status, so think we can just remove this entirely.

https://github.com/thoughtindustries/ti/blob/24bacf0e12d1df4c405a296ad370a9db8b770ca7/assets/javascripts/lms/templates/components/ti-dashboard-access-expandable-item.hbs#L158